dts/bindings/sensor/ti,tmp112.yaml

```diff
@@ -8,3 +8,21 @@ description: |
 compatible: "ti,tmp112"
 
 include: i2c-device.yaml
+
+properties:
+  sampling-rate:
+    description: Sampling rate in mHz (milli-Hertz)
+    type: int
+    default: 1000
```

Comment:
Where does this default come from? Is it the hardware reset value in the sensor?

Reply:
I thought it was but after checking the hardware default is actually 4Hz. I will change that to align with hardware reset values